6-anthos-install/README.md

```diff
@@ -215,132 +231,263 @@ With MCI, we need to select a cluster to be the configuration cluster. In this c
+
+    kubectl create secret generic git-creds --namespace=frontend --context ${CTX_2} --from-file=ssh="${HOME}/.ssh/id_rsa"
+    ```
+
+#### Required Updates
+1. Provide a secret for the application to access CloudSQL.
+
+Bank of Anthos uses two databases, one for the services in the accounts namespace and one for the services in the transactions namespace.
+
+We will assume that the accounts database in the us-west1 region and the transactions database in the us-east1 region. You would need to get the instances' names, and the project ID.
+
+    export SQL_PROJECT_ID=YOUR_SQL_PROJECT_ID
+    export SQL_INSTANCE_NAME_EAST=YOUR_SQL_INSTANCE_NAME_EAST
+    export SQL_INSTANCE_NAME_WEST=YOUR_SQL_INSTANCE_NAME_WEST
+
+Example:
+
+    export SQL_PROJECT_ID=prj-bu1-d-boa-sql-1aec
+    export SQL_INSTANCE_NAME_EAST=boa-sql-1-d-us-east1-65de84c0
+    export SQL_INSTANCE_NAME_WEST=boa-sql-2-d-us-west1-78a54a8f
+
+1. create the CloudSQL secrets
+    ```
+    #Secret for accessing accounts db
+    kubectl create secret generic cloud-sql-admin --context $CTX_1 --namespace=accounts --from-literal connectionName=$SQL_PROJECT_ID:us-west1:$SQL_INSTANCE_NAME_WEST --from-literal=username=admin --from-literal=password=admin
+
+    kubectl create secret generic cloud-sql-admin --context $CTX_2 --namespace=accounts --from-literal connectionName=$SQL_PROJECT_ID:us-west1:$SQL_INSTANCE_NAME_WEST --from-literal=username=admin --from-literal=password=admin
+
+    #Secret for accessing transactions db
+    kubectl create secret generic cloud-sql-admin --context $CTX_1 --namespace=transactions --from-literal connectionName=$SQL_PROJECT_ID:us-east1:$SQL_INSTANCE_NAME_EAST --from-literal=username=admin --from-literal=password=admin
+
+    kubectl create secret generic cloud-sql-admin --context $CTX_2 --namespace=transactions --from-literal connectionName=$SQL_PROJECT_ID:us-east1:$SQL_INSTANCE_NAME_EAST --from-literal=username=admin --from-literal=password=admin
+    ```
+
+1. Add IAM binding
+    ```
+    gcloud iam service-accounts add-iam-policy-binding \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:$GKE_PROJECT_ID.svc.id.goog[accounts/accounts]" \
+    boa-gsa@$GKE_PROJECT_ID.iam.gserviceaccount.com
+
+    gcloud iam service-accounts add-iam-policy-binding \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:$GKE_PROJECT_ID.svc.id.goog[transactions/transactions]" \
+    boa-gsa@$GKE_PROJECT_ID.iam.gserviceaccount.com
+    ```
+
+1. Run script to populate database ledger
+    ```
+    kubectl apply -n transactions --context ${CTX_1} -f ${HOME}/terraform-example-foundation-app/6-anthos-install/db-scripts/populate-ledger-db.yaml
 
-    kubectl apply --context=${CTX_2} -f ${HOME}/terraform-example-foundation-app/acm-config/root-sync.yaml
+    kubectl apply -n transactions --context ${CTX_2} -f ${HOME}/terraform-example-foundation-app/6-anthos-install/db-scripts/populate-ledger-db.yaml
```

Comment:
The file ${HOME}/terraform-example-foundation-app/6-anthos-install/db-scripts/populate-ledger-db.yaml does not exist